pandas/tests/series/test_operators.py

```diff
@@ -1574,33 +1578,38 @@ def test_dt64_series_add_intlike(self, tz):
 
 
 class TestSeriesOperators(TestData):
-    def test_op_method(self):
-        def check(series, other, check_reverse=False):
-            simple_ops = ['add', 'sub', 'mul', 'floordiv', 'truediv', 'pow']
-            if not compat.PY3:
-                simple_ops.append('div')
-
-            for opname in simple_ops:
-                op = getattr(Series, opname)
-
-                if op == 'div':
-                    alt = operator.truediv
-                else:
-                    alt = getattr(operator, opname)
-
-                result = op(series, other)
-                expected = alt(series, other)
-                assert_almost_equal(result, expected)
-                if check_reverse:
-                    rop = getattr(Series, "r" + opname)
-                    result = rop(series, other)
-                    expected = alt(other, series)
-                    assert_almost_equal(result, expected)
+    @pytest.mark.parametrize(
+        'ts',
+        [
+            lambda x: (x, x * 2, False),
```

Comment:
don't write it like this rather split out parameters
```
@pytest.mark.parametrize(
'ts'
[
    (lamda x: x, lamda x: x[::2], False),
....
])
```
its slightly longer but much more obvious

Reply:
Sometimes I look at the results of these format changes and say "yah, he's right, that is more readable."  This isn't one of those times.  Doing it anyway.